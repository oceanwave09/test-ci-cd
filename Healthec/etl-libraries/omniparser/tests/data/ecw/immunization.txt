apuid|RecordID|ItemID|PatientId|PatientAccountNo|EncounterId|ProviderNPI|GivenDate|ImmunizationDescription|HistoryFlag|LotNumber|VaccineManufacturer|ImmunizationDose|Site|Route|ExpirationDate|CVXCode|CPTCode|ImmunizationGivenBy|ImmunizationStatus|VFCStatus
280807|280807.17502.2440559.39571|419352|17502|17502|2440559|1396385019|2022-10-17|A-Hep B|N|K9HS5|GlaxoSmithKline|1 mL|LD|IM Intramuscular||43|90746|9231|Administered|
280807|280807.17502.2517824.39572|445894|17502|17502|2517824|1679623896|2022-10-17|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|546664|Administered|
280807|280807.17502.2540717.39573|445894|17502|17502|2540717|1679623896|2022-10-17|A-Influenza Afluria|N|AS3703F|Seqirus|1 mL||IM Intramuscular||150|90686|9231|Administered|
280807|280807.17502.2527186.39574|445894|17502|17502|2527186|1679623896|2022-10-17|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|546664|Administered|
280807|280807.17502.2531144.39578|445894|17502|17502|2531144|1396385019|2022-10-17|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|546664|Administered|
280807|280807.17502.2531144.39579|419351|17502|17502|2531144|1396385019|2022-10-17|A-Pneumococcal|N|W019533|Merck &Co.|0.5 mL|RD|IM Intramuscular||33|90732|546664|Administered|
280807|280807.17502.2532849.39576|445894|17502|17502|2532849|1396385019|2022-10-17|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|546664|Administered|
280807|280807.17502.2532849.39577|419352|17502|17502|2532849|1396385019|2022-10-17|A-Hep B|N|NT9X3|GlaxoSmithKline|0.5 mL|RD|IM Intramuscular||43|90746|546664|Administered|
280807|280807.17502.2515651.39575|445894|17502|17502|2515651|1841223997|2022-10-17|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|551983|Administered|
280807|280807.17502.2533778.39581|445894|17502|17502|2533778|1467466581|2022-10-18|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|551983|Administered|
280807|280807.17502.2533783.39580|445894|17502|17502|2533783|1467466581|2022-10-18|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|551983|Administered|
280807|280807.17502.2553095.39585|419352|17502|17502|2553095|1396385019|2022-10-18|A-Hep B|N|NT9X3|GlaxoSmithKline|1 mL|RD|IM Intramuscular||43|90746|545291|Administered|
280807|280807.17502.2486511.39586|445894|17502|17502|2486511|1679623896|2022-10-18|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|551459|Administered|
280807|280807.17502.2543020.39588|445894|17502|17502|2543020|1679623896|2022-10-18|A-Influenza Afluria|N|AS3703F|Seqirus|0.5|RD|IM Intramuscular||150|90686|551459|Administered|
280807|280807.17502.2497468.39589|445894|17502|17502|2497468|1679623896|2022-10-18|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|551459|Administered|
280807|280807.17502.2482626.39599|445894|17502|17502|2482626|1679623896|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|552098|Administered|
280807|280807.17502.2493458.39605|445894|17502|17502|2493458|1679623896|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|552098|Administered|
280807|280807.17502.2534702.39597|445894|17502|17502|2534702|1093238222|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|552287|Administered|
280807|280807.17502.2416003.39603|445894|17502|17502|2416003|1093238222|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|552287|Administered|
280807|280807.17502.2550634.39614|445894|17502|17502|2550634|1093238222|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|552287|Administered|
280807|280807.17502.2495274.39598|667972|17502|17502|2495274|1366565202|2022-10-19|A-Influenza Quadrivalent|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||158|90688|545291|Administered|
280807|280807.17502.2541471.39602|667972|17502|17502|2541471|1366565202|2022-10-19|A-Influenza Quadrivalent|N|AS3703F|Seqirus|0.5|RD|IM Intramuscular||158|90688|545291|Administered|
280807|280807.17502.2506940.39612|445894|17502|17502|2506940|1366565202|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5|LD|IM Intramuscular||150|90686|545291|Administered|
280807|280807.17502.2513203.39615|445894|17502|17502|2513203|1366565202|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5|LA|ID Intradermal||150|90686|545291|Administered|
280807|280807.17502.2482607.39613|445894|17502|17502|2482607|1841223997|2022-10-19|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|543617|Administered|
280807|280807.17502.2555200.39622|445894|17502|17502|2555200|1841223997|2022-10-20|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|544271|Administered|
280807|280807.17502.2417113.39616|445894|17502|17502|2417113|1093238222|2022-10-20|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|LD|IM Intramuscular||150|90686|551459|Administered|
280807|280807.17502.2549427.39635|4117502|17502|17502|2549427|1396385019|2022-10-20|A-TDAP|N|YR7AA|GlaxoSmithKline|0.5 mL|RD|IM Intramuscular||115|90715|9231|Administered|
280807|280807.17502.2555690.39647|445894|17502|17502|2555690|1841223997|2022-10-20|A-Influenza Afluria|N|AS3703F|Seqirus|0.5 mL|RD|IM Intramuscular||150|90686|543617|Administered|
