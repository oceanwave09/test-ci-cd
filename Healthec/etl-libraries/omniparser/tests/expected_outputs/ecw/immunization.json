[
	{
		"apuid": "280807",
		"cpt_code": "90746",
		"cvx_code": "43",
		"encounter_id": "2440559",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Hep B",
		"immunization_dose": "1 mL",
		"immunization_given_by": "9231",
		"immunization_status": "Administered",
		"item_id": "419352",
		"lot_number": "K9HS5",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2440559.39571",
		"route": "IM Intramuscular",
		"row_id": "a1d54e58-bf4c-448c-901b-b9ec7140c77c",
		"site": "LD",
		"vaccine_manufacturer": "GlaxoSmithKline"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2517824",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "546664",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2517824.39572",
		"route": "IM Intramuscular",
		"row_id": "41837609-9150-4956-90eb-b53ff3ac077e",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2540717",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "1 mL",
		"immunization_given_by": "9231",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2540717.39573",
		"route": "IM Intramuscular",
		"row_id": "bb686a16-e227-4c1b-a7f1-395e86f3be43",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2527186",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "546664",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2527186.39574",
		"route": "IM Intramuscular",
		"row_id": "d7c55ced-5445-4f14-98a0-24c16f8d8a9b",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2531144",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "546664",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2531144.39578",
		"route": "IM Intramuscular",
		"row_id": "fb3846ec-9918-43ad-ad94-772235a5a6cc",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90732",
		"cvx_code": "33",
		"encounter_id": "2531144",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Pneumococcal",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "546664",
		"immunization_status": "Administered",
		"item_id": "419351",
		"lot_number": "W019533",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2531144.39579",
		"route": "IM Intramuscular",
		"row_id": "0ef61466-c339-4bc0-b3b7-89a1765bead2",
		"site": "RD",
		"vaccine_manufacturer": "Merck &Co."
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2532849",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "546664",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2532849.39576",
		"route": "IM Intramuscular",
		"row_id": "3b3c633a-5db5-4c2e-989a-322f33fac4f2",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90746",
		"cvx_code": "43",
		"encounter_id": "2532849",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Hep B",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "546664",
		"immunization_status": "Administered",
		"item_id": "419352",
		"lot_number": "NT9X3",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2532849.39577",
		"route": "IM Intramuscular",
		"row_id": "14725598-ba3c-4c92-a4c5-3444d72d2526",
		"site": "RD",
		"vaccine_manufacturer": "GlaxoSmithKline"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2515651",
		"given_date": "2022-10-17",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "551983",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"record_id": "280807.17502.2515651.39575",
		"route": "IM Intramuscular",
		"row_id": "d6416bf8-5c23-474b-84ee-41c8981d9ba1",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2533778",
		"given_date": "2022-10-18",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "551983",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "a20c8781a21e3e874d0815e4e4faab60",
		"record_id": "280807.17502.2533778.39581",
		"route": "IM Intramuscular",
		"row_id": "0ff8a9a3-01df-4319-a406-9391f7e06d59",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2533783",
		"given_date": "2022-10-18",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "551983",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "a20c8781a21e3e874d0815e4e4faab60",
		"record_id": "280807.17502.2533783.39580",
		"route": "IM Intramuscular",
		"row_id": "25b95a84-f194-4a75-84ad-54f302946286",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90746",
		"cvx_code": "43",
		"encounter_id": "2553095",
		"given_date": "2022-10-18",
		"history_flag": "N",
		"immunization_description": "A-Hep B",
		"immunization_dose": "1 mL",
		"immunization_given_by": "545291",
		"immunization_status": "Administered",
		"item_id": "419352",
		"lot_number": "NT9X3",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2553095.39585",
		"route": "IM Intramuscular",
		"row_id": "a76ef5fb-4c52-49b2-a730-a7ec2d9225cf",
		"site": "RD",
		"vaccine_manufacturer": "GlaxoSmithKline"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2486511",
		"given_date": "2022-10-18",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "551459",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2486511.39586",
		"route": "IM Intramuscular",
		"row_id": "6c0b89cc-d111-40a6-aa9a-5575411d4e33",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2543020",
		"given_date": "2022-10-18",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5",
		"immunization_given_by": "551459",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2543020.39588",
		"route": "IM Intramuscular",
		"row_id": "025d7f91-bbae-4f1d-919f-db5c2276b6b9",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2497468",
		"given_date": "2022-10-18",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "551459",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2497468.39589",
		"route": "IM Intramuscular",
		"row_id": "b968d901-a56f-4856-9cdf-0c44429e5a48",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2482626",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "552098",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2482626.39599",
		"route": "IM Intramuscular",
		"row_id": "0ab54784-d0e7-4649-9034-3552c9216ed6",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2493458",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "552098",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2493458.39605",
		"route": "IM Intramuscular",
		"row_id": "12713edc-3dcd-4308-8ba9-ef98417d34af",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2534702",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "552287",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e862212969680d9f64910c9b6b6a38d5",
		"record_id": "280807.17502.2534702.39597",
		"route": "IM Intramuscular",
		"row_id": "0139fb55-f3aa-4a3a-ab70-5e4e6caeb08b",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2416003",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "552287",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e862212969680d9f64910c9b6b6a38d5",
		"record_id": "280807.17502.2416003.39603",
		"route": "IM Intramuscular",
		"row_id": "b3c23c72-752d-4760-8ccc-3aea6e019e25",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2550634",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "552287",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e862212969680d9f64910c9b6b6a38d5",
		"record_id": "280807.17502.2550634.39614",
		"route": "IM Intramuscular",
		"row_id": "fd1e13a5-d670-4489-abc1-67be3c3712d1",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90688",
		"cvx_code": "158",
		"encounter_id": "2495274",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Quadrivalent",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "545291",
		"immunization_status": "Administered",
		"item_id": "667972",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "073e509e52f867fad2fef49277008f4a",
		"record_id": "280807.17502.2495274.39598",
		"route": "IM Intramuscular",
		"row_id": "f36c7689-f895-4b5c-86e7-1193b990b8fb",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90688",
		"cvx_code": "158",
		"encounter_id": "2541471",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Quadrivalent",
		"immunization_dose": "0.5",
		"immunization_given_by": "545291",
		"immunization_status": "Administered",
		"item_id": "667972",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "073e509e52f867fad2fef49277008f4a",
		"record_id": "280807.17502.2541471.39602",
		"route": "IM Intramuscular",
		"row_id": "a8477a18-f8a0-4a0a-a2cd-53359a1cc673",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2506940",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5",
		"immunization_given_by": "545291",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "073e509e52f867fad2fef49277008f4a",
		"record_id": "280807.17502.2506940.39612",
		"route": "IM Intramuscular",
		"row_id": "783e0a46-313c-4bd0-af16-6b7ead4f8d4a",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2513203",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5",
		"immunization_given_by": "545291",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "073e509e52f867fad2fef49277008f4a",
		"record_id": "280807.17502.2513203.39615",
		"route": "ID Intradermal",
		"row_id": "1eca831c-55ea-4ece-8cfd-26ceb2033092",
		"site": "LA",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2482607",
		"given_date": "2022-10-19",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "543617",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"record_id": "280807.17502.2482607.39613",
		"route": "IM Intramuscular",
		"row_id": "22718e28-455d-4c82-9ad0-eb00d318fcd0",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2555200",
		"given_date": "2022-10-20",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "544271",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"record_id": "280807.17502.2555200.39622",
		"route": "IM Intramuscular",
		"row_id": "06af8ba0-0ab2-4c7d-8148-afc7bee8d7c7",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2417113",
		"given_date": "2022-10-20",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "551459",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e862212969680d9f64910c9b6b6a38d5",
		"record_id": "280807.17502.2417113.39616",
		"route": "IM Intramuscular",
		"row_id": "8fa51c3b-0a8d-4714-b1da-10da5679fa4d",
		"site": "LD",
		"vaccine_manufacturer": "Seqirus"
	},
	{
		"apuid": "280807",
		"cpt_code": "90715",
		"cvx_code": "115",
		"encounter_id": "2549427",
		"given_date": "2022-10-20",
		"history_flag": "N",
		"immunization_description": "A-TDAP",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "9231",
		"immunization_status": "Administered",
		"item_id": "4117502",
		"lot_number": "YR7AA",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2549427.39635",
		"route": "IM Intramuscular",
		"row_id": "9771b5ab-fe7d-41f1-91a3-50b6cc04c22f",
		"site": "RD",
		"vaccine_manufacturer": "GlaxoSmithKline"
	},
	{
		"apuid": "280807",
		"cpt_code": "90686",
		"cvx_code": "150",
		"encounter_id": "2555690",
		"given_date": "2022-10-20",
		"history_flag": "N",
		"immunization_description": "A-Influenza Afluria",
		"immunization_dose": "0.5 mL",
		"immunization_given_by": "543617",
		"immunization_status": "Administered",
		"item_id": "445894",
		"lot_number": "AS3703F",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"record_id": "280807.17502.2555690.39647",
		"route": "IM Intramuscular",
		"row_id": "984e9436-dcf9-4c80-b008-57a7c0611a50",
		"site": "RD",
		"vaccine_manufacturer": "Seqirus"
	}
]
